app_backend/server.py

```diff
@@ -1,7 +1,50 @@
-from fastapi import FastAPI
+from fastapi import FastAPI, Response, status
 
 app = FastAPI()
 
 @app.get("/")
 async def root():
-    return {"message": "This is the App Backend!"}
+    return {"msg": "This is the App Backend!"}
+
+
+"""
+    add_friend implements the /new-friend route
+
+    resp: Response
+        The response to send back to the user which contains the status code and body
+
+    uuid: str
+        The UUID of the user who is adding the new friend
+
+    fuuid: str
+        The UUID of the friend to be added
+
+    returns Response
+        Response.body: dict
+            Provides any msgs/errs for the request
+        Response.status: int
+            The status code for the request
+"""
+@app.get("/new-friend/", status_code=status.HTTP_201_CREATED)
```

Comment:
This is good. It can also be a post or put request but both implementations work. 

Post is a bit more secure than get as it won't be stored in the browser's history, but for adding friends, I guess it doesn't matter.

Reply:
Nice catch! In my head it was intended as a `POST` request, but I copied the previous function as a template and forgot to change the `GET` to `POST`, Thanks!